Global database and summary sheet.R

```diff
@@ -496,367 +677,337 @@ reliabilitysheet <- reliabilitysheet %>%
-
-# DatabarsconditionalFormatting
-conditionalFormatting(crxls, "riskflags", cols = 17:20, rows = 1:191, type = "databar", style = c("#C6EFCE", "#CD5C5C"))
-conditionalFormatting(crxls, "Reliability_sheet", cols = 2:4, rows = 1:191, type = "databar", style = c("#C6EFCE", "#CD5C5C"))
-conditionalFormatting(crxls, "Alternativeflag_sheet", cols = 21, rows = 1:191, type = "databar", style = c("#C6EFCE", "#CD5C5C"))
-
-#----------------------------------Insert Global Maps---------------------------------------------------------------------
-# install.packages("librarian")     #Run if librarian is not already installed
-librarian::shelf(ggplot2, cowplot, lubridate, rvest, dplyr, viridis, tidyverse, countrycode)
-
-# Loading world database
-world <- map_data("world")
-world <- world %>%
-  dplyr::rename(Country = region) %>%
-  dplyr::mutate(Country = suppressWarnings(countrycode(Country, origin = "country.name", destination = "iso3c")))
-
-# Join datasets with risk flags
-worldmap <- inner_join(world, riskflags, by = "Country")
-
-# Map theme
-plain <- theme(
-  axis.text = element_blank(),
-  axis.line = element_blank(),
-  axis.ticks = element_blank(),
-  panel.border = element_blank(),
-  panel.grid = element_blank(),
-  axis.title = element_blank(),
-  plot.title = element_text(hjust = 0.5),
-  panel.background = element_rect(fill = "#2C3E4F", colour = "#2C3E4F"),
-  plot.background = element_rect(fill = "#2C3E4F", colour = "#2C3E4F"),
-  legend.background = element_rect(fill = "#2C3E4F", colour = "#2C3E4F"),
-  text = element_text(colour = "lightgrey")
-)
-
-# Draw map one
-map <- ggplot(data = worldmap, mapping = aes(x = long, y = lat, group = group)) +
-  coord_fixed(1.3) +
-  geom_polygon(aes(fill = TOTAL_EXISTING_COMPOUND_RISK_SCORE_INCMEDIUM)) +
-  scale_fill_distiller(palette = "Blues", direction = 1) + # or direction=1
-  ggtitle("Total Existing Compound Risk Score") +
-  plain +
-  labs(fill = "Total # of risks")
-
-# Draw map two
-map2 <- ggplot(data = worldmap, mapping = aes(x = long, y = lat, group = group)) +
-  coord_fixed(1.3) +
-  geom_polygon(aes(fill = TOTAL_EMERGING_COMPOUND_RISK_SCORE_INCMEDIUM)) +
-  scale_fill_distiller(palette = "Reds", direction = 1) + # or direction=1
-  ggtitle("Total Emerging Compound Risk Score") +
-  plain +
-  labs(fill = "Total # of risks")
-
-# Join the maps and print to the system
-jointmap <- cowplot::plot_grid(map, map2, ncol = 1, align = c("hv"))
-print(jointmap)
-
-# Insert plot into the worksheet
-insertPlot(crxls, 1, xy = c("AA", 5), width = 11.5, height = 9.5, fileType = "png", units = "in")
-
-# Save plots
-ggsave("Plots/Snapshots/global_emerging_map.pdf", map2, width = 11.5, height = 9.5)
-ggsave("Plots/Snapshots/global_existing_map.pdf", map, width = 11.5, height = 9.5)
-
-#----------------------------------------Save the final worksheet------------------------------------------------------
-saveWorkbook(crxls, file = "Risk_sheets/Compound_Risk_Monitor.xlsx", overwrite = TRUE)
+# 
```

Comment:
The summary excel file is currently produced in a different way; thus this has been commented out. No active code below this point